canonical/specs-v1.canonical.com
# Cache method calls that call Google services

# Done
- Extract the call to `spreadsheet.get_sheet_by_title` to a separate function to ensure query parameters don't cache-bust
- Added `time_cache` decorator
- Decorated `get_sheet_by_title` with 30 min `time_cache`
- Decorated `get_document` with 30 min `time_cache` to decrease calls to Google Drive.

# QA
- Run the project and use it as you would, refresh multiple times
- Wait >30 mins and refresh the page - everything should work as before
